pubspec.yaml

```diff
@@ -4,9 +4,10 @@ author: Clean Team <clean@vacuumlabs.com>
 description: Bindings of the ReactJS library for building interactive interfaces.
 homepage: http://cleandart.github.io/react-dart/
 environment:
-  sdk: '>=0.8.10+6 <2.0.0'
+  sdk: '>=1.13.0 <2.0.0'
```

Comment:
I'd rather include older 1.13.x versions in the range since they should work with these changes; no real reason to exclude them.